windows/runner/flutter_window.cpp

```diff
@@ -6,10 +6,8 @@
 
 #include <optional>
 
-#include "flutter/generated_plugin_registrant.h"
-
-FlutterWindow::FlutterWindow(const flutter::DartProject& project)
-    : project_(project) {}
+FlutterWindow::FlutterWindow(std::shared_ptr<flutter::FlutterEngine> engine)
+    : engine_(std::move(engine)) {}
```

Comment:
TODO: Update the README's engine commit ID once https://github.com/loic-sharma/flutter-engine/pull/46 is merged

Reply:
Unrelated: Any chance as part of that we could also merge in the latest changes from upstream?

Reply:
Yup I can do that after the Windows prototype lands (I'm expecting conflicts in the Windows embedder)